examples/connext_dds/detect_samples_dropped/c++/droppedSamplesExample.idl

```diff
@@ -10,6 +10,6 @@
  * use or inability to use the software.
```

Comment:
In think we're now converging to using PascalCase for file names as well. So you'd have: DroppedSamplesExample.idl